app/services/bulk_upload/lettings/validator.rb

```diff
@@ -73,6 +73,10 @@ def any_logs_already_exist?
     row_parsers.any?(&:log_already_exists?)
   end
 
+  def total_logs_count
```

Comment:
seems odd to me to have this in the validator - is there a better place for it?

Reply:
Yeah, I'm not sure, I think I put it there cause we need the csv parser to get the count and it currently lives in the validator. We also want the update to the db to happen when we are processing the logs and that's why we call it where we do. If you have any thoughts on where this could be moved tho, let me know